### ISSUE TITLE
[FEAT] Support the FeedRegistry on BSC

### ISSUE THREAD

**Issue by wmitsuda** [smartcontractkit/chainlink#6807]
According to the docs, Chainlink currently only supports the FeedRegistry contract on ethereum mainnet and kovan.

Some BSC users of my project would like to see token prices in USD, but it is not possible since there is no feed registry.

**Comment by supportteam99** [comment]
Contact the Support team directly to correct the issue     https://direct.lc.chat/14269734/

**Comment by PatrickAlphaC** [comment]
You can still use the price feeds contracts, but you'd just need to get the contract addresses outside the registry. 

https://docs.chain.link/docs/bnb-chain-addresses/

**Comment by wmitsuda** [comment]
yes, I know that, the point of this issue is to ask for the FeedRegistry to be available on bsc, I don’t want to get the addresses outside the registry and have to hardcode them

could you reopen it?

**Comment by PatrickAlphaC** [comment]
Ah, I see. I'm leaving it closed and adding the "needs votes" label for others to upvote it.
